### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by lsoucasse**
This PR sets up documentation for Mors on readthedocs. Documentation will be available on [https://fwl-mors.readthedocs.io](https://fwl-mors.readthedocs.io).

It also duplicates the download data cli that Stef has just implemented on Janus.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -512,3 +501,23 @@ The Cluster class also contains this function and is called in the exact same wa
...
+- Stellar radius in Rsun, time in yr
+```
+Rstar = baraffe.BaraffeStellarRadius(time)
+```
+- Bolometric flux in Lsun, time in yr
+```
+Lbol = baraffe.BaraffeLuminosity(time)
+```
+- Flux scaled by the star-planet distance in W m<sup>-2</sup>, time in yr, distance in AU
+```Flux = baraffe.BaraffeSolarConstant(time, distance)
```
Comment (stefsmeets) [inline README.md:522]:
````suggestion
```
Flux = baraffe.BaraffeSolarConstant(time, distance)
````

**Review by timlichtenberg — APPROVED** [review]

**Comment by lsoucasse** [comment]
> Nice, looks good to me!
> 
> I searched very hard for any issues, and I could only find a minor formatting issue 😅

Thanks @stefsmeets. I need you to approve it again since I commited your formatting issue.

**Review by stefsmeets — APPROVED** [review]

**Comment by lsoucasse** [comment]
Hi @stefsmeets, the doc website has not been built (https://fwl-mors.readthedocs.io). Are we missing something here?

**Comment by stefsmeets** [comment]
Did you import the project on readthedocs?

**Comment by stefsmeets** [comment]
Ah, I see it was not set up yet, it's available now: https://fwl-mors.readthedocs.io/en/latest/

Can you register on readthedocs and let me know your username?

**Comment by lsoucasse** [comment]
> Ah, I see it was not set up yet, it's available now: https://fwl-mors.readthedocs.io/en/latest/
Thanks!
 
> Can you register on readthedocs and let me know your username?

Done, used Github to sign up so username is "lsoucasse".

**Pull request merged**